README.md

```diff
@@ -139,6 +139,18 @@ bb-fuel supports multiple DBS versions. See below which version maps to the requ
 Note: Starting version 2.4.0 Kubernetes is added as supported platform.
 Note: Starting version 2.0 PCF is added as supported platform.
 
+## Installation
+
+- Install [IntelliJ](https://www.jetbrains.com/idea/) IDE.
+- [Java](https://www.java.com/en/) installed
+- Install and configure [Maven](https://www.jetbrains.com/help/idea/maven-support.html#maven_import_project_start).
```

Comment:
Please link to this page for maven setup: https://backbase.atlassian.net/wiki/spaces/BAT/pages/270696566/Maven+configuration.

Reply:
Please keep in mind there are external developers as well. This maven setup is accessible for our customers:
https://community.backbase.com/documentation/DBS/latest/backbase_ref_maven_configuration